Week-2/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
@relhanakshay I could see that the content is not responsive. Can you check it out. The articles are not wrapping down the image.

Reply:
<img width="475" alt="Screenshot 2022-07-30 at 10 57 52 AM" src="https://user-images.githubusercontent.com/9133593/181876194-9ab65d68-e9b8-4cf8-8fc8-2ed5e63afc66.png">

Reply:
Hi Rajesh, I have made it responsive now.